smtp/client.js

```diff
@@ -85,7 +85,7 @@ Client.prototype =
       if(!(msg instanceof message.Message) 
           && msg.from 
           && (msg.to || msg.cc || msg.bcc)
-          && (msg.text || this._containsInlinedHtml(msg.attachment)))
+          && (msg.text || this._containsInlinedHtml(msg.attachment) || msg.template))
```

Comment:
It's not an extra parenthesis, it's the end of the if condition.
____________________
Luis Rogelio Hernández López
luis.hlopez@outlook.com
@Kurtz1993
Sent using CloudMagic
On Mon, Mar 02, 2015 at 12:06 pm, Narciso <notifications@github.com> wrote:In smtp/client.js:
> @@ -85,7 +85,7 @@ Client.prototype =
>        if(!(msg instanceof message.Message) 
>            && msg.from 
>            && (msg.to || msg.cc || msg.bcc)
> -          && (msg.text || this._containsInlinedHtml(msg.attachment)))
> +          && (msg.text || this._containsInlinedHtml(msg.attachment) || msg.template))

I think you have an extra parenthesis at the end.

  (msg.text || this._containsInlinedHtml(msg.attachment) || msg.template)


—Reply to this email directly or view it on GitHub.